server/functions/src/refreshcasestats.ts

```diff
@@ -0,0 +1,371 @@
+import axios from "axios";
+import * as admin from "firebase-admin";
+import * as functions from "firebase-functions";
+import { SERVING_REGION } from "./config";
+import * as who from "./who";
+
+// THis seems wrong to initializeApp both here and index.ts?
+//admin.initializeApp();
+//const db = admin.firestore();
+// The following line causes the Firebase emulator to say:
+//  "It appears your code is written in Typescript, which must be compiled before emulation."
+//const db = admin.firestore();
```

Comment:
That's puzzling! I... Have no idea why it would say that. 😆

Reply:
See comment above. Maybe you can try/debug on your machine?

Reply:
I'd be very surprised if this issue persists after we move initialization to a separate `db.ts` module. Happy to help debug if it's still the case then.